src/Util.cc

```diff
@@ -214,6 +215,10 @@ std::string entityTypeStr(const Entity &_entity,
   {
     type = "actor";
   }
+  else if (_ecm.Component<components::ParticleEmitter>(_entity))
+  {
+    type = "particle_emitter";
+  }
```

Comment:
update `UtilTest.EntityTypeId` test?

Reply:
Added in 8a6478207e6ebc0522c0fd8e4531a8ce1c0ff7c5